### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by FurkhanAnsari** [Homebrew/discussions#4271] [Getting started]
### Output of `brew config`

```shell
curl: (28) Failed to connect to raw.githubusercontent.com port 443 after 75227 ms: Couldn't connect to server
```

### Output of `brew doctor`

```shell
not installed yet
```

### Description of issue

 this is the error which i get whenever i try. to install brew in my mac how to resolve this i am try to find the ways to resolve but i failed to find 
my internet connection is good and my firewall is turned off help me out

**Comment by gromgit** [comment]
Can you open `https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh` in your browser?

If you can, try running the installer again.

If you can't even open that link in your browser, then you'll have to figure out why you're being blocked.
